addons/repair/XEH_postInit.sqf

```diff
@@ -4,4 +4,17 @@
 [QGVAR(setVehicleHitPointDamage), {_this call FUNC(setHitPointDamage)}] call CBA_fnc_addEventHandler;
 
 // wheels
-[QGVAR(setWheelHitPointDamage), {(_this select 0) setHitPointDamage [_this select 1, _this select 2]}] call CBA_fnc_addEventHandler;
+[QGVAR(setWheelHitPointDamage), {
```

Comment:
We could replace use of this event with `QEGVAR(common,setVanillaHitPointDamage)`